src/server/server_family.h

```diff
@@ -162,6 +162,9 @@ class ServerFamily {
   bool AwaitDispatches(absl::Duration timeout,
                        const std::function<bool(util::Connection*)>& filter);
 
+  // Sets the server to replicate another instance. Used with --replicaof flag.
```

Comment:
Nit: I'd drop where the function is used, it's something readers can look up and it can be picked up by other use cases in the future as well